app/views/idv/capture_doc/selfie.html.erb

```diff
@@ -0,0 +1,30 @@
+<% title t('doc_auth.titles.doc_auth') %>
```

Comment:
Template for hybrid flow was missing

Reply:
This seems like a great candidate for adding a test to catch, are we able to? Or does it require too much because of the JS?

Reply:
Done